lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/mv/CoreFacade.java

```diff
@@ -19,12 +19,9 @@
  */
 package eu.learnpad.mv;
 
-import javax.ws.rs.Path;
-
 import eu.learnpad.mv.rest.ModelImporter;
 import eu.learnpad.mv.rest.NotifyVerification;
 
-@Path("/learnpad/mv/corefacade")

```

Comment:
Don't know why you removed the `Path` from here.  It's the common interface to all REST services for the Core Facade (`ModelImporter`, `NotifyVerification`).